sbpy/data/ephem.py

```diff
@@ -127,14 +128,26 @@ def from_horizons(cls, targetids, id_type='smallbody',
         # modify epoch input to make it work with astroquery.jplhorizons
         # maybe this stuff should really go into that module....
         _epochs = None  # avoid modifying epochs in-place
+        _sorted = False
         if epochs is None:
             _epochs = [Time.now().utc.jd]
         elif isinstance(epochs, Time):
             if epochs.scale != 'utc':
                 warn(('converting {} epochs to utc for use in '
                       'astroquery.jplhorizons').format(epochs.scale),
                      TimeScaleWarning)
-            _epochs = epochs.utc.jd
+            _epochs = epochs.utc.jd  # might this lose some precision?
```

Comment:
I don't think this will lose precision.  [Astropy.time documentation](https://docs.astropy.org/en/stable/time/index.html) says`Time` will maintain sub-ns precision.  You can even specify 16-byte long-double type for output.

Reply:
Long-double output would be great, if it propagates through to the Horizons API, and if Horizons can retain that precision on their end.  Seems like we should have an open issue to investigate this.

Reply:
Agreed.  I can verify the 1 ms precision in my computer, too.  I'm fine with just using the default (double-precision) here.